benches/crypto.rs

```diff
@@ -89,6 +89,17 @@ mod ed25519_benches {
                 .map(|key| key.public().clone())
                 .collect();
 
+            let aggregate_signature =
```

Comment:
Also, please add this at the end of this benchmark (not as a the first one) so we have Ed, BLS_batching_no_aggregation, BLS_aggregated